src/main/java/dynamo/RidesDao.java

```diff
@@ -25,5 +35,52 @@ public RidesDao() {
 
 
 
+	// inserts a new person into the table with an email & a name
+	public void insert(String email, String name, int year, String phoneNumber, String church, boolean attendance) {
+
+		Table table = dynamoDB.getTable("RideTable");
```

Comment:
Let's make "RideTable" a member variable for now since we're gonna need it in each function. In the end we probably want some sort of constructor that correctly initializes which table we want to use (depending on whether we're testing or using the database for reals).